jms/serializer-bundle/2.0/config/packages/jms_serializer.yaml

```diff
@@ -2,12 +2,9 @@ jms_serializer:
     visitors:
         xml:
             format_output: '%kernel.debug%'
-#    metadata:
-#        auto_detection: false
-#        directories:
-#            any-name:
-#                namespace_prefix: "My\\FooBundle"
-#                path: "@MyFooBundle/Resources/config/serializer"
-#            another-name:
-#                namespace_prefix: "My\\BarBundle"
-#                path: "@MyBarBundle/Resources/config/serializer"
+    metadata:
+        auto_detection: true
+        directories:
+            app:
+                namespace_prefix: "App"
+                path: "%kernel.project_dir%/config/serializer"
```

Comment:
Is this the recommended directory to put metadata configurations?

Reply:
Probably the dir name should be `jms_serializer`, to avoid conflicts with the symfony serializer

Reply:
There is no recommended directory for such metadata configurations. But there was a discussion about a place of `translations` directory:

First it was moved to `/config/translations` (see https://github.com/symfony/recipes/pull/249), but then it was moved back to the root of repository (see https://github.com/symfony/recipes/pull/253) because `Discoverability of this directory is important because it's one of those dirs that could be browsed by non backend developers (such as assets/ and templates/).` (c)

I've chosen `config/serilizer` because metadata configuration files are stored in `<bundle-directory>/Resources/config/serializer` for bundles and discoverabilty of `config/serialier` directory is not important as for translations, and because metadata configuration is configuration =)

Reply:
What about  `%kernel.project_dir%/config/metadata/jms_serializer` ?

Reply:
Sounds good!
Cache dir is `%kernel.cache_dir%/jms_serializer` and metadata configuration will be in `%kernel.project_dir%/config/metadata/jms_serializer`. Similar and unique )

Reply:
>Cache dir is `%kernel.cache_dir%/jms_serializer` and metadata configuration will be in > `%kernel.project_dir%/config/metadata/jms_serializer`. Similar and unique )

:+1: